### PR TITLE
Remove Prerender; Instead add some Flavour to load pre-boot

### PR DESCRIPTION
**Pull request by DarkPurple141**

### CODE REVIEW

File: frontend/public/index.html [hunk 1]
```diff
@@ -11,29 +11,138 @@
   <meta property="og:url" content="<%= VUE_APP_URL %>">
   <meta property="og:image" content="/logo.png">
   <title><%= VUE_APP_TITLE %></title>
+  <style>
```
Comment (NunoDasNeves) [inline frontend/public/index.html]: seems like a good idea

File: frontend/public/preload.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  class Loader {
+    constructor(el, root) {
+      this._loaded = 0
+      this._started = 0
+      this.$el = el
+      this.$root = root
+    }
+
+    startLoad() {
+      console.info('starting load')
```
Comment (NunoDasNeves) [inline frontend/public/preload.js]: mebe remove dese

File: frontend/public/preload.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+      this._started += 1
+    }
+
+    endLoad() {
+      console.info('ending load')
```
Comment (NunoDasNeves) [inline frontend/public/preload.js]: remove

File: frontend/src/App.vue [hunk 4]
```diff
@@ -21,6 +21,9 @@ export default {
     }
   },
   created() {
+    [1, 2, 3].forEach(() => {
```
Comment (NunoDasNeves) [inline frontend/src/App.vue]: 3 times..?

Comment (DarkPurple141) [reply frontend/src/App.vue]: Removed this.

**Comment by DarkPurple141** [comment]
> it looks nice but i don't really get why it's done with this _started and _loaded counters and stuff

Explain further?

**Comment by NunoDasNeves** [comment]
No it's ok i get it now

**Pull request merged**